addons/goggles/CfgVehicles.hpp

```diff
@@ -9,6 +9,24 @@ class CfgVehicles {
                     statement = QUOTE(call FUNC(clearGlasses));
                     exceptions[] = {"isNotInside", "isNotSitting", "isNotSwimming", "isNotEscorting"};
                 };
+				class GVAR(respiratorConnect) {
+					displayName = CSTRING(Respirator_Connect);
+					condition = QUOTE(['combo'] call FUNC(respiratorConditions));
```

Comment:
Don't need to create an array for single arg calls
```suggestion
					condition = QUOTE('combo' call FUNC(respiratorConditions));
```

Reply:
Header is fine